src/modules/commander/Commander.cpp

```diff
@@ -2385,6 +2386,32 @@ Commander::run()
 			}
 		}
 
+
+		/* Check for failure detector status */
+		if (armed.armed) {
```

Comment:
&& vehicle_control_mode.flag_control_attitude_enabled?

Reply:
I would like to leave it like this as it is just an absolute attitude check and not an attitude error check (for now).